internal/config/job.go

```diff
@@ -13,13 +14,36 @@ type JobSubmit struct {
 
 	// Flags
 	Filename     string
+	Attach       string
+	Timeout      time.Duration
 	TemplateVals TemplateVals
 }
 
 func (c *JobSubmit) AddFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVarP(&c.Filename, "filename", "f", "", "YAML or JSON file containing the jobs creation request")
 	cmd.MarkFlagRequired("filename")
 	cmd.Flags().Var(&c.TemplateVals, "set", "--set var=val")
+
+	cmd.Flags().StringVarP(&c.Attach, "attach", "", "", "Waits until the job runs exits. Accept stdout or stderr as param")
+
+	cmd.Flags().DurationVar(&c.Timeout, "timeout", 0, "Timeout when waiting any retry or connection fails. If timeout is 0, will wait for completion or cancellation")
```

Comment:
I would rewrite like `timeout when waiting for the job to be started, if 0 is specified, no timeout will be applied and the command will wait until completion or cancellation of the job (default 0)`